src/psacheckbox/psacheckbox.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Checkbox from '@mui/material/Checkbox';
+
+export const PSACheckbox = ({
+  checked,
+  onChange,
+  name,
+  color = 'primary',
+  style,
+  dataTest,
+  comparisonKeys,
+  valueKey,
+  dispatchRedux,
+  updateAction,
```

Comment:
Can you follow the readme file to reduce the props here and use `...props` instead? You can refer to `Tooltip` or `Slider` for reference